app/src/main/java/org/astraea/app/consumer/Builder.java

```diff
@@ -21,29 +21,21 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.HashMap;
-import java.util.HashSet;
 import java.util.Map;
 import java.util.Objects;
-import java.util.Set;
 import org.apache.kafka.clients.consumer.ConsumerConfig;
 import org.apache.kafka.clients.consumer.KafkaConsumer;
 
-public class Builder<Key, Value> {
+abstract class Builder<Key, Value> {
   private final Map<String, Object> configs =
       new HashMap<>(
           Map.of(ConsumerConfig.GROUP_ID_CONFIG, "groupId-" + System.currentTimeMillis()));
```

Comment:
Only `TopicBuilder` needs to initialize group id.

Reply:
Forgot to remove this line, many thanks~